tests/comp_options.js

```diff
@@ -30,7 +30,7 @@ message Message {
 tape.test("complex options", function (test) {
   var root = protobuf.parse(proto).root;
 
-  test.deepEqual(root.parsedOptions[0], {
+  test.deepEqual(root.parsedOptions[1], {
```

Comment:
Since we now have defaults as the first parsed option, this test needs to look for the second parsed option.

Reply:
`parsedOptions` are just options that have been properly parsed - in other words, a step after options. Since features are just options, and they need to be parsed just like options, they do need to go through this step. If you want I can delete them from options after they've been copied over into _features.

Reply:
My confusion is why *defaults* end up here.  Parsed features make sense to me, but you're explicitly adding edition defaults to `parsedOptions`

Reply:
Gotcha - yeah it seems it is related. I think the point is that we're taking advantage of the parsing `setParsedOptions` offers, which ultimately is needed in feature resolution. If we move it outside of setting parsedOptions into some other feature resolution-specific method, we're essentially copying over reusable code. Same reason why we're caling [setParsedOption here](https://github.com/protobufjs/protobuf.js/pull/2029/files/087a33e526f61cbf38af9b1c76342bcd06ba394b#diff-42f95033dbb9c5068c9cc85ce7c08cf1c22f68199a26bfab67817c63ff804253R638) too, since it takes advantage of all the parsing already offered in that method.